extensions/datadog_statsd.go

```diff
@@ -104,6 +104,11 @@ func (s *StatsD) HandleNotificationFailure(game string, platform string, err *er
 	s.Client.Incr("failed", []string{fmt.Sprintf("platform:%s", platform), fmt.Sprintf("game:%s", game), fmt.Sprintf("reason:%s", err.Key)}, 1)
 }
 
+//InitializeFailure notifu error when is impossible tho initilizer an app
+func (s *StatsD) InitializeFailure(game string, platform string) {
+	s.Client.Incr("failed", []string{fmt.Sprintf("platform:%s", platform), fmt.Sprintf("game:%s", game)}, 1)
```

Comment:
If we're incrementing `failed` metric aren't we mixing it with notifications failures? We should use another name such as `initialize_failure`.

Reply:
I copy from the previous function and didn't notice that.